tests.py

```diff
@@ -318,7 +319,7 @@ def test_logger_name_is_test(self):
     def test_path_name_is_test(self):
         logger.error('An error has occured')
         json_record = json.loads(log_buffer.getvalue())
-        self.assertIn('json-log-formatter/tests.py', json_record['pathname'])
+        self.assertIn(os.path.basename(os.path.abspath('.')) + '/tests.py', json_record['pathname'])
```

Comment:
Could you please tell when does this test break and how can it be reproduced?

Reply:
Just by not having the name of the root directory of the project to be "json-log-formatter".

Reply:
There are no plans to rename the directory name, so I am not sure how it can help.

Reply:
For example, i made a debian package python3-json-log-formatter
The source of that package, when being built in debian infrastructure, is put in some directory like
`python-json-log-formatter_0.5.2/`
Since the tests are run from that directory, one of them fail.
The change I propose here let the test work independently of the name of the parent directory where the test is run.

Reply:
I see, that makes sense.